addons/mrp/models/mrp_workorder.py

```diff
@@ -125,6 +125,8 @@ def name_get(self):
     @api.one
     @api.depends('production_id.product_qty', 'qty_produced')
     def _compute_is_produced(self):
+        if not self.production_id:
+            return
```

Comment:
As @sle-odoo said, a workorder should always have a production_id. A solution would be to use a default_production_id